rest/models/aggs.go

```diff
@@ -49,25 +47,21 @@ type AggsResponse struct {
 	Aggs         []Aggregate `json:"results,omitempty"`
 }
 
-// GetAggsParams is the set of path and query parameters that can be used when requesting aggs through the Get method.
+// GetAggsParams is the set of path and query parameters for requesting aggs.
 type GetAggsParams struct {
-	Ticker      string
-	Multiplier  int
-	Resolution  Resolution
-	From        time.Time
-	To          time.Time
-	QueryParams GetAggsQueryParams
-}
+	Ticker     string     `validate:"required"`
```

Comment:
I already like this more than having an embedded `QueryParams` struct. 

The user should only have to care about what's required, and not thinking:
 "if it's in the path then it's required, if it's in the `QueryParams` it's not required. Also the path is everything in `GetAggsParams` except for the embedded `QueryParams`. Seeing `required` explicitly listed here saves a lot of thinking from the user.